src/main/webapp/creatives/maps/constants.js

```diff
@@ -0,0 +1,11 @@
+
+/** A list of server routes that keep track of each interaction when fetched. */
+export const Urls = {
```

Comment:
Could you add the Urls constant to the existing constants.js file that lives in the creatives folder?

Reply:
oh yes, good call!